supplementary_style_guide/style_guidelines/code-commands.adoc

```diff
@@ -10,3 +10,5 @@
 
 [[long-code-examples]]
 == Long code examples in procedures
+
+All code blocks, regardless of length, should be necessary, accurate, and helpful. Code blocks should be as copy-and-paste friendly as possible, with the exception of variables and callouts only. The length of the block is irrelevant, within reason, if the code block follows these guidelines. For more information about code sample suggestions, see IBM Style Guide pp. 198-200.
```

Comment:
```suggestion
All code blocks, regardless of length, must be necessary, accurate, and helpful. Code blocks must be as copy-and-paste friendly as possible, with the exception of variables and callouts only. The length of the block is irrelevant, within reason, if the code block follows these guidelines. For more information about code sample suggestions, see IBM Style Guide pp. 198-200.
```

Suggestion to make the 'should's into 'must's for IBM style guidance about being direct.

Reply:
@ndeevy made these updates here